application/ui/widgets/eh-widgets-dir/eh-widgets/src/page/bundle-group/BundleGroupPage.js

```diff
@@ -121,14 +127,28 @@ const BundleGroupPage = () => {
                           alt="BundleGroup Logo" />}
                     </div>
 
-                    {(pageModel.bundleGroup.displayContactUrl) &&
-                      <>
-                        <div className="BundleGroupPage-contact-us">
-                          <p>{i18n.t('page.bundleGroupInfo.contactUsInfo')}</p>
-                          <Button href={pageModel.bundleGroup.contactUrl} target="_blank">{i18n.t('component.button.contactUs')}</Button>
-                        </div>
-                        <hr/>
-                      </>
+                    {(pageModel.bundleGroup.displayContactUrl) && (pageModel.bundleGroup.contactUrl)
```

Comment:
@anasliferay This logic isn't quite right. I'm getting a Contact Us modal on ALL URLs now when displayContactUrl and contactUrl are not set because the else always executes.

Reply:
@nshaw updated the condition to not display the CONTACT US button at all when there is no contact URL based on length.